common/utils/loginUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { v4 as uuid } from 'uuid';
+
+export const generateRandomUser = () => {
```

Comment:
Those a not loginUtils.ts

it is generateRandomUser.ts or generateRandomUser.util.ts

Reply:
or generate-random-user.ts or generate_random_user.ts